binr/rasm2/rasm2.c

```diff
@@ -124,9 +121,11 @@ static int showanal(RAsmState *as, RAnalOp *op, ut64 offset, ut8 *buf, int len)
 static int show_analinfo(RAsmState *as, const char *arg, ut64 offset) {
 	ut8 *buf = (ut8 *)strdup ((const char *)arg);
 	int ret, len = r_hex_str2bin ((char *)buf, buf);
+	PJ *pj = pj_new ();
```

Comment:
Add a check if !pj, check other part that uses pj for examples.

Reply:
Added the null check

Reply:
Feel free to click the "resolve conversation" when you fix something